test/sql/types/bit/test_bit_functions.test

```diff
@@ -0,0 +1,177 @@
+# name: test/sql/types/bit/test_bit_functions.test
+# description: BIT tests
+# group: [bit]
+
+statement ok
+PRAGMA enable_verification
+
+statement ok
+CREATE TABLE bits (b bit);
+
+# insert valid bit strings
+statement ok
+INSERT INTO bits VALUES('101001111'), ('00111'),  ('100101010110000000000001'), ('111111010100')
```

Comment:
Could we add a persistent storage test as well? See e.g. `test/sql/storage/test_storage_scan.test`

Reply:
Done!